### DISCUSSION TITLE
HA e Alexa con starlink

### DISCUSSION THREAD

**Discussion by vebto53** [indomus/forum#3088] [HUB: Home Assistant]
Ciao, ho home assistant OS su raspberry 4 . Ho installato starlink che non ha indirizzi pubblici e ha una rete tutta sua, Ho installato clouflare e tramite questo l'accesso a Home Assistant da remoto funziona bene. Quello che mi succede e che non avevo previsto e che Alexa non "vede" più le entità Home Assistant integrate tramite skill haaska per intenderci. la skill risulta dall'app di alexa collegata, ma penso di doverla in qualche modo reindirizzare nel tunnel cloudflare, ammesso che ciò sia fattibile. Qualcuno ha già affrontato il problema. Grazie

**Comment by KC-inDomus** [comment]
https://github.com/indomus/forum/discussions/3074
